airflow/roles/airflow/tasks/main.yml

```diff
@@ -4,23 +4,67 @@
     name: airflow-stable
     repo_url: https://airflow-helm.github.io/charts
 
+# create in line sealed secret since ansible variables don't propagate  all the way to the k8s manifests
+- name: Create secret for database connection
+  community.kubernetes.k8s:
+    state: present
+    namespace: airflow
+    definition:
+      apiVersion: bitnami.com/v1alpha1
+      kind: SealedSecret
```

Comment:
Do these need to be per env sealed?

Reply:
I believe so. This sealed secret will be to connect to the postgres database and I am assuming (and hoping) we have a different instance per environment.